src/wrappers/themis/wasm/src/secure_cell_seal.ts

```diff
@@ -83,12 +83,10 @@ export class SecureCellSeal {
         "message must be not empty"
       );
     }
-
-    let context;
-    if (arguments.length > 1 && arguments[1] !== null) {
-      context = coerceToBytes(arguments[1]);
-    } else {
+    if (context == null) {
       context = new Uint8Array();
+    } else {
+      context = coerceToBytes(context);
     }
```

Comment:
Why exactly is `undefined` check not needed here but necessary over there?

Reply:
+1

Reply:
Added if ( context === null || context === undefined).

Reply:
Now `decrypt()` is a bit of an oddball as it does not have this check.

But I believe it does not need it since it's the only method which – for some reason – had the signature
```typescript
decrypt(message: Uint8Array, context: Uint8Array = new Uint8Array())
```
all along from the first published version, and thus explicitly passing `undefined` there is an error. We have to allow `undefined` in other places, as that's what JS – when TypeScript is not used – would *actually* pass when only one argument is used. Right?

What about `null` context in decrypt() then? Why doesn't it need a special condition? Is it allowed at all?